src/smpp_gateway/router.py

```diff
@@ -0,0 +1,35 @@
+from rapidsms.messages.incoming import IncomingMessage
+from rapidsms.messages.outgoing import OutgoingMessage
+from rapidsms.router.blocking import BlockingRouter
+
+from smpp_gateway.models import MTMessage
+
+
+class PriorityIncomingMessage(IncomingMessage):
+    def respond(self, text, **kwargs):
+        fields = kwargs.get("fields", {})
+        if "priority_flag" not in fields:
+            fields["priority_flag"] = MTMessage.PriorityFlag.LEVEL_2.value
+        kwargs["fields"] = fields
+        return super().respond(text, **kwargs)
+
+
+class PriorityOutgoingMessage(OutgoingMessage):
+    def extra_backend_context(self):
+        context = super().extra_backend_context()
+        context["priority_flag"] = self.fields.get(
+            "priority_flag", MTMessage.PriorityFlag.LEVEL_2.value
```

Comment:
Does this need to be `LEVEL_2` to meet our other goals with this PR?

If not, perhaps we could set the a regular outgoing message (non-response) to `LEVEL_1` ("normal") priority (also as a class variable to make it easier to override in a project), keeping `LEVEL_2` for responses to mobile-originated messages?